src/assets/styles/components/_article.scss

```
@@ -16,7 +16,8 @@
     width: 100%;
   }
 
-  &__cover-image {
+  &__cover-image,
+  &__cover-video {
```

Тут пока нет смысла для двух разных селекторов:

```suggestion
  &__cover-media {
```

Аналогично поменять в шаблоне.